HISTORY.md

```diff
@@ -1,13 +1,17 @@
 # Version history
 
-## 1.3.0-beta.2 /2019-11-29
+## 1.3.0-beta.2 /2019-12-06
 * Update node dependencies.
 * The `@std/esm` package has been deprecated in favour of `esm`. Also `esm` have limited the `.mjs` files to basic functionality without support for esm options, to align with the current experimental support in Node. [More details here](https://github.com/standard-things/esm/issues/696). According to that all `.mjs` files in the `romcal` project have been renamed back to `.js`.
 * Locales management and pickup in romcal have been refactored, and it now behaves like Moment.js (used by romcal) or the general approach for i18n in app development: locale containing region like `fr-CA` will gracefully fall back to `fr` if a localisation key isn't in `fr-CA` or if `fr-CA.js` doesn't exits in the `src/locales` directory. In the end, it always falls back to `en`, which is the default language in romcal. Full explanations have been added to the `README.md`.
 * Locale files have been renamed to kebab-case with capitalized regions to follow the IETF Language Codes standards and be close to the general conventions in software development for i18n.
+* In locale files, the `general` and `national` parts are now merged in a new `sanctoral` part. This will avoid potential duplicated or missing keys between these 2 parts, particularly when a celebration is move from the general to the national calendar. Also, keys in celebrations and the new sanctoral have been sorted alphabetically. ([#97](https://github.com/romcal/romcal/pull/97))
+* Romcal in Italian ! New `calendars/italy.js` and `locales/it.js` added.
```

Comment:
I would reformulated this. Maybe just simply state that _Itallian translation has been added_. Also, you probably want to link the PR #94 and say that @gtortone has aided you. :smiley:

Reply:
Right, i forgot to add the link on that one!